src/components/activity-page/activity-page-content.tsx

```diff
@@ -48,14 +47,21 @@ export class ActivityPageContent extends React.PureComponent <IProps, IState> {
     const vertical = page.layout === PageLayouts.FullWidth;
     const primaryFirst = page.layout === PageLayouts.FullWidth || page.layout === PageLayouts.FortySixty;
 
+    const introEmbeddables = page.embeddables.filter((e: any) => e.section === EmbeddableSections.Introduction);
     const primaryEmbeddables = page.embeddables.filter((e: any) => e.section === EmbeddableSections.Interactive);
     const secondaryEmbeddables = page.embeddables.filter((e: any) => !e.section);
```

Comment:
I'm guessing this is correct, but just wondering if it would be safer to filter all those whose section is not `Introduction` or `Interactive`, in case we ever start naming secondary sections.

Reply:
Ah, that's a good idea.  I'm not entirely comfortable with the section being null to represent that an embeddable goes in the info/assessment section.  Your idea seems much safer.